sdk/src/main/kotlin/io/rover/sdk/ui/blocks/concerns/layout/BlockViewModel.kt

```diff
@@ -145,7 +146,7 @@ internal class BlockViewModel(
     override val events = eventSource.share()
 
     override val isClickable: Boolean
-    get() = block.tapBehavior !is Block.TapBehavior.None
+    get() = this.block is ButtonBlock || this.block.tapBehavior != Block.TapBehavior.None
```

Comment:
[PP/Bike shedding]  I prefer not to use the `this` keyword when not necessary but personal preference